src/tests/ftest/pool/info_tests.py

```diff
@@ -21,128 +21,60 @@
   Any reproduction of computer software, computer software documentation, or
   portions thereof marked with this legend must also reproduce the markings.
 """
-import os
-import json
-from apricot import Test
+from apricot import TestWithServers
+from test_utils import TestPool
 
 
-import agent_utils
-import server_utils
-import write_host_file
-from daos_api import DaosContext, DaosPool, DaosLog
-from conversion import c_uuid_to_str
+class InfoTests(TestWithServers):
+    """Test class for pool query.
+
+    Test Class Description:
+        This test verifies destruction of a pool that is rebuilding.
 
-class InfoTests(Test):
-    """
-    Tests DAOS pool query.
     :avocado: recursive
     """
-    def setUp(self):
-        self.agent_sessions = None
-        # get paths from the build_vars generated by build
-        with open(os.path.join(os.path.dirname(os.path.realpath(__file__)),
-                               "../../../../.build_vars.json")) as build_file:
-            build_paths = json.load(build_file)
-        self.basepath = os.path.normpath(build_paths['PREFIX'] + "/../")
-        self.server_group = self.params.get("name", '/server_config/',
-                                            'daos_server')
 
-        context = DaosContext(build_paths['PREFIX'] + '/lib/')
+    def test_pool_info_query(self):
+        """Jira ID: DAOS-xxxx.
 
-        self.pool = DaosPool(context)
-        self.d_log = DaosLog(context)
-        self.hostlist_servers = self.params.get("test_machines1",
-                                                '/run/hosts/')
-        self.hostfile_servers = write_host_file.write_host_file(
-            self.hostlist_servers, self.workdir)
-        self.agent_sessions = agent_utils.run_agent(self.basepath,
-                                                    self.hostlist_servers)
-        server_utils.run_server(self.hostfile_servers, self.server_group,
-                                self.basepath)
+        Test Description:
+            Create and connect to a pool.  Verify the pool information returned
+            from the pool query matches the values used to create the pool.
 
-    def tearDown(self):
-        # shut 'er down
-        try:
-            if self.pool:
-                self.pool.destroy(1)
-            os.remove(self.hostfile_servers)
-        finally:
-            if self.agent_sessions:
-                agent_utils.stop_agent(self.agent_sessions)
-            server_utils.stop_server(hosts=self.hostlist_servers)
-
-    def test_simple_query(self):
-        """
-        Test querying a pool created on a single server.
+        Use Cases:
+            Verify pool query.
 
-        :avocado: tags=pool,poolquery,infotest
+        :avocado: tags=all,tiny,pr,pool,smoke,infotest
```

Comment:
```suggestion
        :avocado: tags=all,tiny,pr,pool,smoke,poolinfoquery
```
Would be a better globally unique name, IMHO.